clients/src/main/java/org/apache/kafka/clients/CommonClientConfigs.java

```diff
@@ -63,6 +63,9 @@
     public static final String METRICS_NUM_SAMPLES_CONFIG = "metrics.num.samples";
     public static final String METRICS_NUM_SAMPLES_DOC = "The number of samples maintained to compute metrics.";
 
+    public static final String METRICS_RECORD_LEVEL_CONFIG = "metrics.record.level";
+    public static final String METRICS_RECORD_LEVEL_DOC = "The highest recording level for metrics.";
```

Comment:
Do we need to expose a similar config on the broker side since the broker also uses the client side metrics in certain cases?